client/state/first-view/actions.js

```diff
@@ -0,0 +1,61 @@
+/**
+ * External dependencies
+ */
+import moment from 'moment';
+
+/**
+ * Internal dependencies
+ */
+import {
+	FIRST_VIEW_EMBRACE,
+	FIRST_VIEW_DISMISS,
+	FIRST_VIEW_HIDE,
+	FIRST_VIEW_SHOW,
+} from 'state/action-types';
+
+import { FIRST_VIEW_START_DATES } from './constants';
+
+export function init( { user } ) {
+	const userCreated = moment( user.get().date );
+	let eligibleViews = [];
+
+	for ( let view in FIRST_VIEW_START_DATES ) {
+		if ( moment( FIRST_VIEW_START_DATES[ view ] ).isBefore( userCreated ) ) {
+			eligibleViews.push( view );
+		}
+	}
+
+	return {
+		type: FIRST_VIEW_SHOW,
+		views: eligibleViews,
+	};
+}
+
+export function dismissView( { view } ) {
+	return {
+		type: FIRST_VIEW_DISMISS,
+		view,
+	};
+}
+
+export function embraceView( { view } ) {
```

Comment:
Basically, it's "forgetting a dismissal". I could, I suppose, just have dismiss take a parameter that is either true or false... but otherwise, "embrace" was the best I could do naming-wise ;)

I'm not really sure if there's benefit to having them be two separate actions, but it felt weird to me having `dismissed` (bool) as a parameter to `dismissView`. Thoughts?